requirements_all.txt

```diff
@@ -219,6 +219,9 @@ blockchain==1.4.4
 # homeassistant.components.sensor.bme680
 # bme680==1.0.5
 
+# homeassistant.components.camera.bomradarcam
```

Comment:
Move, `git mv`, the camera platform under a new bomradarcam component package. That's our new way of structuring modules.
```
homeassistant/components/bomradarcam/__init__.py
homeassistant/components/bomradarcam/camera.py
```

Make sure to update `.coveragerc` and run `script/gen_requirements_all.py` after the move.